### ISSUE TITLE
CreatingClientSecretsFile

### ISSUE THREAD

**Issue by EPSteve** [jay0lee/GAM#75]
Steps do not match what I am seeing on google pages:
https://console.developers.google.com/project/.....
e.g.
API and Auth screens do not match
I don't see a "Create another client ID" button
running "gam oauth info" doesn't display a Client ID

**Comment by GI-thub** [comment]
Can you share a screenshot of what you do see?


 Graham Ingleby | Cloud Computing Consultant | 01344 203395 |
gingleby@ancoris.com
[image: Google+] <http://google.com/+Ancoris>
<http://www.linkedin.com/company/98835>  <http://www.ancoris.com>

On 26 June 2015 at 17:52, EPSteve <notifications@github.com> wrote:

> Steps do not match what I am seeing on google pages:
> https://console.developers.google.com/project/.....
> e.g.
> API and Auth screens do not match
> I don't see a "Create another client ID" button
> running "gam oauth info" doesn't display a Client ID
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jay0lee/GAM/issues/75>.
>

-- 


------------------------------

This message is for the named person's use only.  If you receive this message in error, please delete it and notify the sender.  Ancoris reserves the right to monitor all e-mail communications through its networks. Ancoris Limited, Registered in England Number: 04830784, Registered address: 5a Frascati Way, Maidenhead, Berkshire SL6 4UY. Trading Address: Lily Hill House, Lily Hill Road, Bracknell, Berkshire RG12 2SJ

------------------------------

**Comment by EPSteve** [comment]
I have now been able to complete the instructions at 
 https://github.com/jay0lee/GAM/wiki/CreatingClientSecretsFile
 
As it turns out, there just a few screens that look a bit different.  
There are also a few different choices ... see below
 

 
No Groups Settings API, Site Verification API

 
No Client ID displayed
 
gam oauth info
Select the authorized scopes for this token. Include a 'r' to grant read-only
access or an 'a' to grant action-only access.
 
[*]  0)  Group Directory API (supports read-only)
[*]  1)  Organizational Unit Directory API (supports read-only)
[*]  2)  User Directory API (supports read-only)
[*]  3)  Chrome OS Device Directory API (supports read-only)
[*]  4)  Mobile Device Directory API (supports read-only and action)
[*]  5)  User Email Settings API
[*]  6)  Calendar Resources API
[*]  7)  Audit Monitors, Activity and Mailbox Exports API
[*]  8)  Admin Settings API
[*]  9)  Groups Settings API
[*] 10)  Calendar Data API (supports read-only)
[*] 11)  Audit Reports API
[*] 12)  Usage Reports API
[*] 13)  Drive API (create report documents for admin user only)
[*] 14)  License Manager API
[*] 15)  User Security Directory API
[*] 16)  Notifications Directory API
[*] 17)  Site Verification API
(*) 18)  IMAP/SMTP Access (send notifications to admin)
(*) 19)  User Schemas (supports read-only)
 
     20)  Select all scopes
     21)  Unselect all scopes
     22)  Continue
 
 
In client_secrets.json .JSON File
"client_id":"583499324581-b358jr4ovf5la8qmvjico98ec5diahog.apps.googleusercontent.com"
 
Next, the instructions say:
Creating Your Own OAuth2service.json
 
Starting from the end of the previous step after creating your client_secrets.json file, 
Below the section for the Installed applications, click on Create another client ID…
 
Mine just says "Create new Client ID"
which, since my original query, I have now clicked
 

 
I then get  
 

 
rather than
https://camo.githubusercontent.com/7cf1005fb9d5667272720e8bb414f069b9af2cb3/68747470733a2f2f676f6f676c6564726976652e636f6d2f686f73742f3042396c746c6135564f49342d62575a4857586c6a624856774e44412f67616d31342e706e67
 
 
Regards,
Steve
 
Steve Gilbert, CEM®
EBCP™, BPI® Certified: Envelope, 
Heating Professional, & MuiltiFamily
  

(646) 373-1023 (Mobile)
(347) 767-6353 (Office)
(718) 569-6848 (Fax)
 <mailto:Steve@EnergyParadiso.com> Steve@EnergyParadiso.com
 
 <http://www.energyparadiso.com/> www.EnergyParadiso.com
CEM® (Certified Energy Manager) is a Registered
Trademark of the Association of Energy Engineers
 
EBCP™ (Existing Buildings Commissioning Professional) is a
Trademark of the Association of Energy Engineers
 
BPI® is a Registered Trademark of the 
Building Perfomance Institute.
